index.html

```diff
@@ -605,7 +626,7 @@ <h1>Trust Model</h1>
 repository</a>, and
         </li>
         <li>
-the <a>issuer</a> does not need to trust the <a>verifier</a>.
+the <a>issuer</a> does not need to trust (or even know) the <a>verifier</a>.
```

Comment:
Try to avoid the parenthetical... maybe "does not need to trust or know"...

Reply:
Will revise on Monday.